server/src/modules/mentorship-request/mentorship-request.model.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  Model,
+  Column,
+  Table,
+  DataType,
+  ForeignKey,
+  PrimaryKey,
+  AutoIncrement,
+} from 'sequelize-typescript';
+import { User } from '../user/user.model';
+
+@Table({
+  tableName: 'mentorship_request',
+})
+export class MentorshipRequest extends Model<MentorshipRequest> {
+  private static PENDING = 'Pending';
+  private static RESOLVED = 'Resolved';
+
+  private static MENTORSHIP_REQUEST_STATUS = DataType.ENUM(
```

Comment:
Are you saying we can get `mentorshipRequest` status from response? So we don't need this status here?

Reply:
Yes exactly: it's definitely possible to get the status by checking to see if a response exists; I guess the question is whether that's good enough? If checking for a response is slow or inconvenient it's fine to add an enum, but otherwise we probably don't need it. (Not a big deal though: probably not super hard to remove or add it later if necessary.)